platforms/ide/tooling-api/src/main/java/org/gradle/tooling/internal/consumer/connection/ToolingParameterProxy.java

```diff
@@ -26,7 +31,7 @@
  *
  * <p>These proxies do not support any kind of nesting.
  */
-public class ToolingParameterProxy implements InvocationHandler {
+public class ToolingParameterProxy implements InvocationHandler, Serializable {
```

Comment:
❓ Why does it need to be `Serializable` now?

Reply:
Fixed